README.md

```diff
@@ -224,6 +224,39 @@ If your repo has a different command to run for migrations, you can configure it
 `DeclareSchema.db_migrate_command = "bundle exec rails db:migrate_immediate"`
 ```
 
+## The `belongs_to` Relation and Indexes
+By default, `declare_schema` creates an index for `belongs_to` relations. If an index
```

Comment:
I'd start with a higher level sentence than this. Something like:
```
`declare_schema` intercepts the `belongs_to` macro to infer the foreign key column in a model that "points to" the primary key of the associated model. Note that `belongs_to` is outside the `declare_schema do` block.
```
I just remembered that we should have a separate section like this one for `has_and_belongs_to_many`. In that case `declare_schema` infers the intersection table with its foreign keys back to the two associated models.

Reply:
Section rewritten, thank you. Please let me know if you'd like more in the new habtm section.